bin/spring

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
+  require 'rubygems'
+  require 'bundler'
+
+  if match = Bundler.default_lockfile.read
+    .match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
```

Comment:
__[rubocop]__
* Align the operands of a condition in an `if` statement spanning multiple lines.